crates/router/src/core/webhooks/utils.rs

```diff
@@ -4,7 +4,13 @@ use crate::{
 };
 
 fn default_webhook_config() -> api::MerchantWebhookConfig {
-    std::collections::HashSet::from([api::IncomingWebhookEvent::PaymentIntentSuccess])
+    std::collections::HashSet::from([
+        api::IncomingWebhookEvent::PaymentIntentSuccess,
+        api::IncomingWebhookEvent::PaymentIntentFailure,
+        api::IncomingWebhookEvent::PaymentIntentProcessing,
```

Comment:
why we need to handle webhooks for processing status?

Reply:
When payment is created , we will mark the status as authentication_pending and once payment is made, we will get a payment_processing webhook from their side which notifies that the payment is done by customer by not verifed by the network, so now we have change the authentication_pending state to processing state, otherwise it will be in authentication_pending state and customer will make another payment again.